docs/Configuration.md

```diff
@@ -32,6 +32,28 @@ explanation consists of:
 
 The sections below contain links to the lists of available exclude reasons for each type of exclude.
 
+#### Excluding Paths
+
+Path excludes are used to mark a complete path as excluded. They are defined using a
+[glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob), a `reason` and a `comment`:
+
+```yaml
+excludes:
+  paths:
+  - pattern: "test-data/**"
+    reason: "TEST_TOOL_OF"
+    comment: "This folder contains examples which are not distributed."
+```
+
+The path exclude above has the following effects:
+
+* All projects found below the `test-data` directory are marked as excluded.
+* License findings in files below the `test-data` directory are marked as excluded. This can be used to ignore such
+  license findings when writing [evaluator rules](GettingStarted.md#6-running-the-evaluator).
```

Comment:
I'm not sure I fully get what you intend to say in relation to evaluator rules here. Do you want to say

> This mechanism can be used to easily ignore license findings in evaluator rules by checking whether they are excluded.

?

Reply:
Done.